setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='scrapylib',
-    version='1.5.0',
+    version='1.5.1',
```

Comment:
sorry I didn't noticed this early, but the correct way to increase the version of scrapylib package is by using [bumpversion](https://pypi.python.org/pypi/bumpversion) tool.